wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/probe/server/ExecServerLivenessProbeConfigFactory.java

```diff
@@ -23,10 +29,15 @@
  * @author Alexander Garagatyi
  */
 public class ExecServerLivenessProbeConfigFactory implements HttpProbeConfigFactory {
+
   private final int successThreshold;
+  private final MachineTokenProvider machineTokenProvider;
 
-  public ExecServerLivenessProbeConfigFactory(int successThreshold) {
+  @Inject
+  public ExecServerLivenessProbeConfigFactory(
```

Comment:
I think we are able not to change current approach with public available liveness probes.
@skabashnyuk WDYT? Should we revert it here and create an issue for implementing exclude in JWT proxy?